ovos_plugin_manager/vad.py

```diff
@@ -121,7 +121,10 @@ def create(config=None):
             raise ValueError(f"VAD Plugin not configured in: {vad_config}")
         try:
             clazz = OVOSVADFactory.get_class(vad_config)
-            return clazz(vad_config)
+            # module name not expected in config; don't change passed config
+            plugin_config = dict(vad_config)
+            plugin_config.pop("module")
```

Comment:
if "module" always present? shouldnt this pop be conditional?

Reply:
it is always present by the time we get here; if it's not then a `ValueError` will have been raised already